File: 2571-find-the-pivot-integer/2571-find-the-pivot-integer.py
class Solution:
    def pivotInteger(self, n: int) -> int:
        s=0
        for i in range(1,n+1):
            s+=i
        c=0
        for i in range(1,n+1):
            if((c+i)==(s-c)):
                return i
            c+=i
            
        return -1
        

File: 2571-find-the-pivot-integer/2571-find-the-pivot-integer.cpp
class Solution {
public:
    int pivotInteger(int n) {
        int k=0;
        for (int i=1;i<=n;i++){
            k+=i;
        }
        int c=0;
        for(int i=1;i<=n;i++){
            if((c+i)==(k-c)){
                return i;
            }
            c+=i;
        }
        return -1;
        
    }
};